marginaleffects/hypotheses_joint.py

```diff
@@ -10,9 +9,6 @@
 def joint_hypotheses(obj, joint_index=None, joint_test="f", hypothesis=0):
     assert joint_test in ["f", "chisq"], "`joint_test` must be `f` or `chisq`"
 
-    if isinstance(obj, pd.DataFrame):
-        obj = pl.DataFrame(obj)
-
```

Comment:
it seems like `obj` is of type `model` and it shouldn't ever be of type `dataframe`. It was introduced [here](https://github.com/vincentarelbundock/pymarginaleffects/pull/94/files#diff-ebbff9ec0fb9835ce04a0130f6b95c24ce225b77660dccfa85d14fecd402504d) #94  but not sure why. 

If it were to be a dataframe all the following operations would fail so I think we can remove it. @vincentarelbundock do you agree?

Reply:
That seems right but I won't have time to dig deep, so I'll trust you here.